optuna/storages/rdb/storage.py

```diff
@@ -450,9 +448,28 @@ def get_all_study_summaries(self) -> List[StudySummary]:
     def create_new_trial(self, study_id, template_trial=None):
         # type: (int, Optional[FrozenTrial]) -> int
 
+        return self._create_new_trial_with_trial(study_id, template_trial)[0]
+
+    def _create_new_trial_with_trial(
+        self, study_id: int, template_trial: Optional[FrozenTrial] = None
+    ) -> Tuple[int, FrozenTrial]:
+        """Create a new trial and returns trial_id and the trial converted into a FrozenTrial.
```

Comment:
It is a bit difficult for me to parse this sentence. If possible, could you rephrase it? The following is just an example.
```suggestion
        """Create a new trial and return trial_id with the trial converted into a :class:`~optuna.trial.FrozenTrial`.
```

Reply:
I think we can replace it with
```Create a new trial and return trial_id with a :class:`~optuna.trial.FrozenTrial`.```

Reply:
I confirmed the change in  https://github.com/optuna/optuna/pull/1140/commits/5ae39047691fcb01e1a6ac8956049ab76beb5a13.
It looks simpler and easier to understand. Thanks!